docs/marketplace/add_to_marketplace/add_item_by_type/add_custom_resource.md

```diff
@@ -0,0 +1,254 @@
+---
+id: add_custom_resource
+title: Publish your Custom Resources to the Marketplace
+sidebar_label: Create a Custom Resource
+---
+
+when working in the design section of a project you can define and configure custom resources, such resources may be created from the marketplace and you can add your own Custom Resources to the marketplace, this is how..
+
+When designing your project in the Mia-Platform Console, you can define and configure [Custom Resources](/console/design-your-projects/custom-resources.md), either from scratch or from the existing resources already included in the Marketplace. Here we explain how you can add your own Custom Resources to the Marketplace.
+
+They are basically defined by two fields:
+
+- the `apiVersion` field, which is the API version of the resource
+- the `kind` field, which is the kind of the resource
```

Comment:
Wouldn't go into detail here since this page is not intended to explain k8s terminology, but I see the redundancy and I've added a couple of words. See if you like it.